packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.tsx

```diff
@@ -78,12 +79,25 @@ export type DescriptionConfig = {
   showFullDescriptionByDefault?: boolean
 }
 
+export type MapValueToReactComponentConfig = {
+  // The column to apply the mapping to
+  matchColumnName: string
+  Component: React.ComponentType<{
+    /* The value of the table cell. If the column is a _LIST type, then the LIST will be parsed and this will be a string array. */
+    value: string | string[] | null
+    selectColumn: SelectColumn
+  }>
+  // discriminator
+  isMapValueToReactNodeConfig: true
+}
```

Comment:
Super generic interface. We could make _n_ reusable components to get passed into this interface instead of having _n_ different configurations